rs/bitcoin/checker/tests/tests.rs

```diff
@@ -764,3 +834,38 @@ fn assert_reply(result: WasmResult) -> Vec<u8> {
         }
     }
 }
+
+pub struct MetricsAssert {
+    metrics: Vec<String>,
+}
+
+impl MetricsAssert {
```

Comment:
nit: could be potentially nice to share this with the same code in the minter tests through some sort of test fixtures.

Reply:
I agree, for this we would need a new test library that can handle both state machine tests or pocket IC (probably behind a feature flag) but this I would do in a separate ticket, so I opened [XC-260](https://dfinity.atlassian.net/browse/XC-260) for this.